### ISSUE TITLE
dont do it (it will destroy ur account)

### ISSUE THREAD

**Issue by packmastermc** [mrmojololo/blooket-admin-hack#240]
guys seriously this guy is trying to get u to put ur usrname so he can take ur account and also btw the "infinite tokens" is impossible because the maximium tokens possible to be earned in a day is 500 no matter if u admin or whatever

**Comment by mrmojololo** [comment]
this isnt true, you cant hack people from their usernames only

**Comment by packmastermc** [comment]
Yes I understand but isn't it a little suspicious that ur just asking for
username I guarantee this is fake

**Comment by mrmojololo** [comment]
Yes, it is fake but the reason we are collecting usernames is so we can
collect them and create a database

**Comment by orangemonkey01** [comment]
a database for what

**Comment by packmastermc** [comment]
exposed lol fake

**Comment by mrmojololo** [comment]
The database is for collecting usernames so we can log them. Blooket does
not have usernames public at the moment, so I am going to make a list of
every username here so someone can make their own websites with them, maybe
a stats site? Also, this hack is so obviously fake, its a troll to fuck
with people.

**Comment by ThEgLiTc** [comment]
thats so dog bro

**Comment by TOESHAHA** [comment]
> this isnt true, you cant hack people from their usernames only
> […](#)
> Message ID: ***@***.***>

how do we use the admin hack

**Comment by orangemonkey01** [comment]
its fake

**Comment by CryptoDude3** [comment]
the best part abt these is the # of ppl that think its real

**Comment by mrmojololo** [comment]
💀
